### ISSUE TITLE
Please add _netdev... or we need to figure out another way to delay

### ISSUE THREAD

**Issue by ylluminate** [OpenMediaVault-Plugin-Developers/openmediavault-mergerfsfolders#7]
Please see the following thread. Happy to help in anyway possible to properly resolve this since I feel like the `_netdev` is a hackish "solution":
https://forum.openmediavault.org/index.php?thread/39867-mergerfs-creates-fstab-trouble-stops-omv-from-booting-with-root-rw/

**Comment by ryecoaaron** [comment]
Waiting for network devices is not the right solution.  x-systemd.requires should do the right thing.  You just don't have that setup in your hacked up setup.  Please give me some time to fix this.  I'm planning to move to systemd mount files in the 6.x verion and I won't be backporting that to 5.x.

**Comment by ylluminate** [comment]
Thanks @ryecoaaron 🙇🏻‍♂️  I appreciate your awareness and putting this on your todo list. It is a hassle to have to keep modifying `/etc/fstab` each time now - I've moved the MergerFS line out of the OMV section of fstab so that I retain the line and it just simply inserts a duplicate I have to remove each time now I make any mount changes. The config should remain relatively static after I populate it with external drives initially (using these vs networking due to attached speed being a little faster than ethernet atm), so it's not like a crisis. I'm just glad I ascertained a functional workaround that delays long enough to allow ZFS, etc. to load.
